redash/query_runner/google_spreadsheets.py

```diff
@@ -100,18 +100,28 @@ def __init__(self, worksheet_title):
 def parse_query(query):
     values = query.split("|")
     key = values[0]  # key of the spreadsheet
-    worksheet_num_or_title = 0  # A default value for when a number of inputs is invalid
+    worksheet_num = 0  # A default value for when a number of inputs is invalid
```

Comment:
This is what the third bullet point in #6409 is referring to - are these sheets 0-indexed or 1-indexed? We should verify this.

Reply:
Oh I get it now , as far as I know , google sheets start from 1 and then 2,3,... so on , so shall we have default changed to 1